### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'runner'

### ISSUE THREAD

**Issue by telanus** [lutris/lutris#510]
Got this:

```
Traceback (most recent call last):
  File "/home/user01/src/lutris/lutris/gui/config_dialogs.py", line 361, in on_runner_changed
    if self.game.runner:
AttributeError: 'NoneType' object has no attribute 'runner'
```

what I did was to add a game choose libretro, then browse for a rom, and this gets dumped on the terminal

**Comment by daniel-j** [comment]
Fixed in #506

**Comment by strycore** [comment]
Fix merged in master
